neo/rawio/tdtrawio.py

```diff
@@ -197,7 +197,8 @@ def _parse_header(self):
                 for seg_index, segment_name in enumerate(segment_names):
                     # get data index
                     tsq = self._tsq[seg_index]
-                    mask = (tsq['evtype'] == EVTYPE_STREAM) & \
+                    mask = ((tsq['evtype'] == EVTYPE_STREAM) | \
```

Comment:
Do you have an idea why the Stream code in your system is different? In any case the new code should be defined on the class level as the current `EVTYPE_STREAM`.

Reply:
I am not sure why it is different to be honest. What's puzzling me is that the 'TankEvType' in the TBK file is the default EVTYPE_STREAM (or 33025) but the data themselves in the tsq files have an 'evtype' of 33041 for my neural data streams. Therefore, just changing the value of the current EVTYPE_STREAM will not work. I could add a class EVTYPE_STREAM_2 or something similar and check if evtype is equal to either of the EVTYPE_STREAM variables?